src/cljs/athens/hotkeys.cljs

```diff
@@ -0,0 +1,75 @@
+(ns athens.hotkeys
+  (:require
+    ["react-hotkeys" :refer [GlobalHotKeys HotKeys configure]]
+    [re-frame.core :refer [dispatch subscribe]]
+    [athens.util :as util]
+    [athens.router :as router]))
+
+(configure #js{:ignoreTags #js[]})
+
+(def modkey
+  (let [os (util/get-os)]
+    (or (and (= os :mac) "meta")
+      (and (= os :windows) "ctrl")
+      (and (= os :linux) "ctrl"))))
+
+(defn with-mod
+  [combination]
+  (str modkey "+" combination))
+
+(defn not-editing
+  []
+  (nil? @(subscribe [:editing/uid])))
+
+(defn selected-items-not-empty
+  []
+  (not-empty @(subscribe [:selected/items])))
+
+(defn maybe-undo-or-redo
+  [operation]
+  (when
+    (or (not-editing)
+      (selected-items-not-empty))
+    (dispatch [operation])))
+
+(defn global-hotkeys
+  []
+  [:> GlobalHotKeys
+   {:keyMap   {:athena-toggle        (with-mod "k")
```

Comment:
Maybe we could use `athens.config` to store information about key choices.
Current config only reads goog closure defines, store some config in `localStorage` and retrieve it at startup, so users could redefine keybindings.

Reply:
Yeah that is on my roadmap. Allowing to change these at runtime and storing them somewhere (localStorage sounds good).